runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1573,6 +1604,30 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 		memoryParameters[opt_Xmox] = memoryParameters[opt_Xmo];
 	}
 
+	result = option_set_to_opt_double(vm, OPT_XXMAXRAMPERCENT, &index, EXACT_MEMORY_MATCH, &extensions->maxRAMPercent);
+	if (OPTION_OK != result) {
+		goto _error;
+	}
+	memoryParameters[opt_maxRAMPercent] = index;
+	if (memoryParameters[opt_maxRAMPercent] != -1) {
+		if (extensions->maxRAMPercent > 100) {
+			result = OPTION_OVERFLOW;
```

Comment:
Can we reuse J9NLS_GC_OPTIONS_PERCENT_OUT_OF_RANGE here? The only issue I see is current output format is too few digits after decimal mark:
J9NLS_GC_OPTIONS_PERCENT_OUT_OF_RANGE=%1$s value must be between %2$.2f and %3$.2f (inclusive)